src/ATen/native/xpu/sycl/ResizeKernel.cpp

```diff
@@ -25,7 +25,7 @@ void resize_bytes_xpu(StorageImpl* storage, size_t size_bytes) {
   c10::xpu::XPUGuard guard(device.index());
   at::DataPtr data = allocator->allocate(size_bytes);
   if (storage->data_ptr()) {
-    at::globalContext().lazyInitXPU();
+    at::globalContext().lazyInitDevice(c10::DeviceType::XPU);
```

Comment:
I suppose this line is invalid. @fengyuan14 , @guangyey , why do we require `lazyInitDevice`?

Reply:
I agree with your point of view. `lazyInitXPU` should already be called in [`_lazy_init`](https://github.com/pytorch/pytorch/blob/e14b58ffbd4758fb37a4118efb3f01e6a47e0ff2/torch/xpu/__init__.py#L120).

Reply:
Thank you, I will remove this unnecessary init right now